src/test/java/teammates/ui/webapi/DeleteNotificationActionTest.java

```diff
@@ -0,0 +1,69 @@
+package teammates.ui.webapi;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.NotificationAttributes;
+import teammates.common.util.Const;
+import teammates.ui.output.MessageOutput;
+
+/**
+ * SUT: {@link DeleteNotificationAction}.
+ */
+public class DeleteNotificationActionTest extends BaseActionTest<DeleteNotificationAction> {
+    private static final String TEST_NOTIFICATION = "notification1";
+    private final NotificationAttributes testNotificationAttribute = typicalBundle.notifications.get(TEST_NOTIFICATION);
+
+    @Override
+    String getActionUri() {
+        return Const.ResourceURIs.NOTIFICATION;
+    }
+
+    @Override
+    String getRequestMethod() {
+        return DELETE;
+    }
+
+    @Test
+    @Override
+    protected void testExecute() throws Exception {
+        loginAsAdmin();
+
+        ______TS("Typical Case: Delete notification successfully");
+        String[] requestParams = new String[] {
+                Const.ParamsNames.NOTIFICATION_ID, testNotificationAttribute.getNotificationId(),
+        };
+
+        DeleteNotificationAction action = getAction(requestParams);
+        JsonResult response = getJsonResult(action);
+        MessageOutput msg = (MessageOutput) response.getOutput();
+        assertEquals("Notification has been deleted.", msg.getMessage());
+
+        verifyAbsentInDatabase(testNotificationAttribute);
+
+        ______TS("Notification ID cannot be null");
+        requestParams = new String[] {
+                Const.ParamsNames.NOTIFICATION_ID, null,
+        };
+
+        verifyHttpParameterFailure(requestParams);
+
+        ______TS("Deleting non-existent notification");
+        // Deleting a non-existent notificiation will not throw an error and fail silently
+        requestParams = new String[] {
+                Const.ParamsNames.NOTIFICATION_ID, "non-existent notification",
+        };
+
+        verifyAbsentInDatabase(testNotificationAttribute);
```

Comment:
Seems that this line is redundant since the absence of the test notification is not really relevant to this particular test.

Reply:
I wanted to ensure that the notification is not in the data base, to be certain that deleting a non-existent notification would cause a silent failure. Though it is unlikely that someone would add a notification with ID `non-existent notification` to the test bundle

Reply:
If I'm not wrong, `testNotificationAttribute` has `notification1` as the ID and not `non-existent notification`, so it seems that `non-existent notification` should be tested here instead?

Reply:
I see what you mean now. Will verify that a notification with ID `non-existent notification` is absent from the database